examples/expandos/index.js

```diff
@@ -1,6 +1,6 @@
 import createVirtualEnvironment from '@locker/near-membrane-dom';
 
-const evalScript = createVirtualEnvironment();
+const evalScript = createVirtualEnvironment(window, { globalThis: window });
```

Comment:
Could `globalThis` default to the first argument?

Reply:
it doesn't default to the first argument, but to the current globalThis.

Reply:
I'm getting more uncomfortable with the default behavior, this is a low level library, maybe we should just remove all those default behaviors, in which case consumers of this library should pass the specific values for all 3

Reply:
@caridy that's certainly fine with me. I'll do it in a follow up